codemods/jscodeshift/next/13/remove-get-static-props/index.ts

```diff
@@ -172,6 +172,26 @@ export const addRevalidateVariableDeclaration: ModFunction<any, 'write'> = (
 	root,
 	settings,
 ) => {
+	const exportNamedDeclarationAlreadyExists =
+		root.find(j.ExportNamedDeclaration, {
+			declaration: {
+				declarations: [
+					{
+						type: 'VariableDeclarator',
+						id: {
+							type: 'Identifier',
+							name: 'revalidate',
+						},
+					},
+				],
+				kind: 'const',
```

Comment:
it doesn't have to be const? or actually should it be const?

Reply:
i think it should not be const